src/views/product-tutorials-view/server.ts

```diff
@@ -35,11 +39,98 @@ export interface ProductPageData {
 		blocks: ProductViewBlock[]
 		showProductSitemap?: boolean
 	}
-	allCollections: ProductTutorialsSitemapProps['collections']
+	/**
+	 * Collections to render in the product sitemap.
+	 * Required if pageData.showProductSitemap is set to true,
+	 * Can safely be omitted if showProductSitemap is false or not provided.
+	 *
+	 * TODO: should we refactor slightly to avoid this type interdependency?
+	 * Specifically: in server-side getStaticProps, we could only pass
+	 * allCollections to the view IF showProductSitemap is set to true,
+	 * We'd no longer pass pageData.showProductSitemap, and instead render
+	 * the sitemap conditionally based on the presence of allCollections
+	 * (which we could also rename to sitemapCollections for clarity).
+	 *
+	 * (Note: I'm avoiding this for now, since possible redesign
+	 * of /hcp/tutorials may make this refactor irrelevant).
+	 */
+	allCollections?: ProductTutorialsSitemapProps['collections']
 	inlineCollections: InlineCollections
 	inlineTutorials: InlineTutorials
 }
 
+/**
+ * Note: this is a temporary spike to get the /hcp/tutorials rendering.
+ * It relies on fallback content from Waypoint.
+ *
+ * TODO: figure out what to do with the /hcp/tutorials view (design dependent).
+ * Taking this temporary approach for now while awaiting final designs.
+ */
+export async function getCloudTutorialsViewProps() {
+	const productData = cachedGetProductData('hcp')
+	/**
+	 * Grab fallback props (Waypoint's), so that the view can render
+	 * even if there are gaps in what HCP content we can provide
+	 */
+	const tempFallbackProps = JSON.parse(
+		fs.readFileSync(
+			path.join(
+				process.cwd(),
+				'src/views/product-tutorials-view/server-temp-hcp-fallback-content.json'
+			),
+			'utf8'
+		)
+	)
+	/**
+	 * Get the raw page data
+	 */
+	const {
+		pageData: rawPageData,
+		inlineCollections,
+		inlineTutorials,
+	} = await getProductPageContent(ThemeOption.cloud)
+	/**
+	 * Process page data, reformatting as needed.
+	 * Includes parsing headings, for use with the page's sidecar
+	 */
+	const { pageData, headings } = await processPageData(rawPageData)
+
+	return {
+		props: stripUndefinedProperties({
+			metadata: {
+				title: 'Tutorials',
+			},
+			data: {
+				pageData,
+				inlineCollections,
+				inlineTutorials,
+			},
+			layoutProps: {
+				headings,
+				breadcrumbLinks: getTutorialsBreadcrumb({
+					product: { name: productData.name, filename: productData.slug },
+				}),
+				sidebarSections: tempFallbackProps.layoutProps.sidebarSections, // TODO: stop using Waypoint props as temp fallback
```

Comment:
We can use the new section filter to get this data! Hit me up and we can chat through it. 😄